komodo/release_transpiler.py

```diff
@@ -249,5 +264,17 @@ def main():
     args.func(args)
 
 
+def valid_file(path: str) -> str:
+    if os.path.isfile(path):
+        return path
+    raise TypeError(path)
```

Comment:
This is not necessarily a TypeError, it is more likely it is a missing file. There is probably a FileNotFoundErrorException that can be used.